bakery/src/tests/test_cli.js

```diff
@@ -174,6 +174,17 @@ test('stable flow in pdf and distribution pipeline', async t => {
   const assembleMetaResult = await completion(assembleMeta)
   t.truthy(fs.existsSync(`${outputDir}/${bookId}/assembled-book-metadata/${bookId}/collection.assembled-metadata.json`), formatSubprocessOutput(assembleMetaResult))
 
+  const link = spawn('node', [
+    'src/cli/execute.js',
+    ...commonArgs,
+    '--image=localhost:5000/openstax/cops-bakery-scripts:test',
+    'link',
+    bookId,
+    'src/scripts/canonical-book-list.json'
```

Comment:
I'm confused -- why are we trying to passing this when the task is grabbing from the files in the container via `/code/scripts/canonical-book-list.json`?  🤷‍♂️